tests/dialects/test_scf.py

```diff
@@ -1,9 +1,10 @@
-from typing import cast
-
+import pytest
+from typing import cast, List
```

Comment:
nit: prefer `list` over `List`

Reply:
Fixed :) I don't need List now that we are using the builder so I have removed it from the import :)